### DISCUSSION TITLE
To allow mailscanner converting email checking both subject and body

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#16662] [Feature improvement]
When I respond to initial email from customer, this sent email is related to the ticket on the basis of ticket number in the subject. 
This OK however breaking thread in customer email client who initiate email conversation.
This is due to mended email subject.
How about allowing mailscanner searching also body of email for ticket number?

best

**Comment by rskrzypczak** [comment]
There's something like that.

config/Modules/OSSMailScanner.php
```php
   /** Search prefix in body, type: boolean */
   public static $searchPrefixInBody = false;

**Reply by s2b-git** [reply]
This works. Thanks, however... config files can possibly be overwritten while upgrade?

**Reply by rskrzypczak** [reply]
No. Configuration remain unchanged after upgrade.
